### PR TITLE
Light theme & navigation setup

### PR DESCRIPTION
**Pull request by revomhere**

### CODE REVIEW

File: docs/how-to-guides/how-to-guides.mdx [hunk 1]
```diff
@@ -1,9 +1,14 @@
 ---
 image: /img/how-to-guides-og.png
+sidebar_position: 1
 ---
 
 # How-To Guides
 
+```
+yarn install
+```
```
Comment (ardier16) [inline docs/how-to-guides/how-to-guides.mdx:10]: remove

**Review by ardier16 — CHANGES_REQUESTED** [review]

File: docs/tutorials/swapping-tokens.mdx [hunk 2]
```diff
@@ -14,33 +15,43 @@ For more details and limitations about swaps, see [Bridging Protocol](../overvie
 To set up a token swap, create a Rarimo transaction and specify the token and amount that you want to receive, as in this example:
 
 ```ts
-import { ChainNames } from '@rarimo/shared'
-import { CheckoutOperationParams, createCheckoutOperation, EVMOperation, Price, BridgeChain } from '@rarimo/nft-checkout'
-import { createProvider, ProviderUserRejectedRequest } from '@rarimo/provider'
-import { MetamaskProvider } from '@rarimo/providers-evm'
-import { ethers } from "ethers"
+import { ChainNames } from "@rarimo/shared";
```
Comment (ardier16) [inline docs/tutorials/swapping-tokens.mdx:18]: format with prettier

File: docs/tutorials/creating-transactions.mdx [hunk 3]
```diff
@@ -21,86 +22,90 @@ Sending a transaction with the SDK includes these main steps:
...
-This object represents the transaction operation and provides methods to access a specific type of chain, estimate the transaction price, and complete the checkout process.
+   This object represents the transaction operation and provides methods to access a specific type of chain, estimate the transaction price, and complete the checkout process.
 1. Bundle the data for the contracts to call as part of the transaction.
 1. Run the transaction operation's `checkout()` method, which starts the transaction.
 
 Here is an example of creating a transaction on the Goerli test chain:
 
 ```ts
-import { ChainNames, BridgeChain } from '@rarimo/shared'
+import { ChainNames, BridgeChain } from "@rarimo/shared";
```
Comment (ardier16) [inline docs/tutorials/creating-transactions.mdx:32]: all ts code blocks

File: docs/reference/_category_.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "label": "Technical Reference",
+  "position": 6,
+  "link": {}
+}
+
```
Comment (ardier16) [inline docs/reference/_category_.json:6]: remove spaces in the end of all category files
```suggestion

```

File: docs/overview/identity.mdx [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
-
+sidebar_position: 2
+title: Identity Protocol
```
Comment (ardier16) [inline docs/overview/identity.mdx:3]: isn't it defined by h1 title already?

File: docs/overview/bridge.mdx [hunk 6]
```diff
@@ -1,11 +1,14 @@
 ---
+sidebar_position: 3
+title: Bridging Protocol
```
Comment (ardier16) [inline docs/overview/bridge.mdx:3]: same for all other titles

File: docs/overview/_category_.json [hunk 7]
```diff
@@ -0,0 +1,6 @@
+{
+  "label": "About Rarimo",
```
Comment (ardier16) [inline docs/overview/_category_.json:2]:
```suggestion
  "label": "Overview",
```

File: docs/reference/contracts/_category_.json [hunk 8]
```diff
@@ -0,0 +1,5 @@
+{
+  "label": "Conracts",
```
Comment (ardier16) [inline docs/reference/contracts/_category_.json:2]:
```suggestion
  "label": "Contracts",
```

File: docs/use-cases/_category_.json [hunk 9]
```diff
@@ -0,0 +1,6 @@
+{
+  "label": "Overview of Use Cases",
```
Comment (ardier16) [inline docs/use-cases/_category_.json:2]:
```suggestion
  "label": "Use Cases",
```

**Review by ardier16 — APPROVED** [review]

File: docs/overview/identity.mdx [hunk 10]
```diff
@@ -1,5 +1,5 @@
 ---
-
+sidebar_position: 2
```
Comment (ihordiachenko) [inline docs/overview/identity.mdx:2]: Manually specifying the position in each document sucks. Can we list the documents in `sidebars.js` in the desired order?

**Comment by ihordiachenko** [comment]
@revomhere @ardier16 we should detect the theme preference on user's device if available. Currently it's supported at least on Mac OS and Windows

**Comment by ihordiachenko** [comment]
The documents are too narrow on 15" MBP. The text needs some breathing room

image

File: docs/how-to-guides/validator-setup.mdx [hunk 11]
```diff
@@ -9,93 +9,96 @@ We will also set up a broadcaster service, TSS service, and EVM identity saver s
...
 
 1. Connect to `34.66.205.183`.
-This IP address is used for submitting TXs and p2p connections.
+   This IP address is used for submitting TXs and p2p connections.
 
 1. Connect to the RPC node: `39072210913bb52a6444543ad160b04668d58210@34.66.205.183:26656`
 
 1. Execute script to set environments:
 
-  ```bash
```
Comment (ihordiachenko) [inline docs/how-to-guides/validator-setup.mdx:27]: The linter broke markdown. Please re-check the nested lists

image

**Review by ihordiachenko — CHANGES_REQUESTED** [review]

**Review by ihordiachenko — APPROVED** [review]

**Pull request merged**